### PR TITLE
Update require_pulp_plugins

### PR DESCRIPTION
**Pull request by fao89**
[noissue]

### CODE REVIEW

File: pulp_file/tests/functional/utils.py [hunk 1]
```diff
@@ -40,7 +40,7 @@
 def set_up_module():
     """Skip tests Pulp 3 isn't under test or if pulp-file isn't installed."""
     require_pulp_3(SkipTest)
```
Comment (mdellweg) [inline pulp_file/tests/functional/utils.py:42]: can this ever be the case?

File: pulp_file/tests/functional/utils.py [hunk 2]
```diff
@@ -40,7 +40,7 @@
...
-    require_pulp_plugins({"pulp_file"}, SkipTest)
+    require_pulp_plugins({"file"}, SkipTest)
```
Comment (mdellweg) [inline pulp_file/tests/functional/utils.py:43]: This is comparing with the status endpoint?

Comment (mdellweg) [reply pulp_file/tests/functional/utils.py:43]: Yes it is (The doc string there looks outdated, too):

https://github.com/pulp/pulp-smash/blob/45aa7cd9fb7562b304b9af26968b03f22708ecdc/pulp_smash/pulp3/utils.py#L56

**Review by mdellweg — APPROVED** [review]

File: pulp_file/tests/functional/utils.py (hunk 2 above)
Comment (fao89) [reply pulp_file/tests/functional/utils.py:43]: I got this on my dev environment:
```
(pulp) [vagrant@pulp3-source-fedora33 pulp_file]$ pytest -v pulp_file/tests/functional/
==================================================================================== test session starts =====================================================================================
platform linux -- Python 3.9.2, pytest-6.2.2, py-1.10.0, pluggy-0.13.1 -- /usr/local/lib/pulp/bin/python3
cachedir: .pytest_cache
rootdir: /home/vagrant/devel/pulp_file
collected 41 items                                                                                                                                                                           

pulp_file/tests/functional/api/test_crud_content_unit.py::ContentUnitTestCase::test_01_create_content_unit SKIPPED (The following plugins are required but not installed: {'pulp_f...) [  2%]
pulp_file/tests/functional/api/test_crud_content_unit.py::ContentUnitTestCase::test_02_read_content_unit SKIPPED (The following plugins are required but not installed: {'pulp_file'}) [  4%]
pulp_file/tests/functional/api/test_crud_content_unit.py::ContentUnitTestCase::test_02_read_content_units SKIPPED (The following plugins are required but not installed: {'pulp_fi...) [  7%]
```

**Comment by fao89** [comment]
@pulp/core please merge it!

**Review by bmbouter — APPROVED** [review]
We need this fixed. Thanks @fao89 !

**Pull request merged**